### DISCUSSION TITLE
numpydoc style parameters and attributes wrongly uppercased

### DISCUSSION THREAD

**Discussion by apple-phi** [pradyunsg/furo#200] [Potential Issue] [answered]
I'm writing API documentation in the numpydoc style, and using numpydoc to auto-document my code .

I'm not using sphinx.ext.napoleon since it does not format attributes nicely if you have too many. Apart from that, when I use napoleon the formatting of the classes is correct.

However, when I use numpydoc and build for HTML, Furo uppercases all parameters and attributes, which for obvious reasons is not helpful for documentation.

Is there any way I can fix this?

Here's a snippet of my documentation and the corresponding webpage:
```python
"""
A basic FASTA parser with miRBase built in.

Extracts main FASTA sub-components.

Parameters
----------
text : str
    The FASTA text to parse.

Attributes
----------
ids : List[str]
seqs : List[str]
headers : List[str]
descriptions : List[str]
num : int
text : str
source : set, defaults to set()
line_length : int, defaults to 70

Notes
-----
Upon instantiation, the :attr:`text` attribute is formatted
using the :attr:`line_length` attribute.
"""
```
<img width="682" alt="Screenshot 2021-07-22 at 16 17 39" src="https://user-images.githubusercontent.com/65281038/126664136-1de1bc56-d5dc-4ff7-90a2-e23b1b5d7d32.png">

This does not happen with any other html_theme I've tried, only Furo :(

**Comment by pradyunsg** [comment, marked as answer]
Please provide a reproducer for this, with clear instructions on how to generate the documentation. Or provide a link to the generated docs.
